release notes/v0.48.0.md

```diff
@@ -191,10 +194,11 @@ import { browser, networkProfiles } from 'k6/experimental/browser';
 ## UX improvements and enhancements
 
 - [browser#1074](https://github.com/grafana/xk6-browser/pull/1074) Adds a new `browser.closeContext()` [method](https://k6.io/docs/javascript-api/k6-experimental/browser/closecontext/) to facilitate closing the current active browser context.
-- [#3370](https://github.com/grafana/k6/pull/3370). The k6 got a new flag `--profiling-enabled` which enables exposing [pprof](https://go.dev/blog/pprof) profiling endpoints. The profiling endpoints are exposed on the same port as the HTTP REST API under the `/debug/pprof/` path. This could be useful for the extension developers.
-- [#3442](https://github.com/grafana/k6/pull/3442). The k6 now supports a new `--version` flag. It has the same output as `k6 version` command. Thanks, @ffapitalle!
-- [#3423](https://github.com/grafana/k6/pull/3423). The InfluxDB output got an environment variable `K6_INFLUXDB_PROXY` which allows specifying proxy. Thanks, @IvanovOleg!
-- [#3398](https://github.com/grafana/k6/pull/3398). The k6 cloud traces are enabled by default.
+- [#3370](https://github.com/grafana/k6/pull/3370) The k6 got a new flag `--profiling-enabled` which enables exposing [pprof](https://go.dev/blog/pprof) profiling endpoints. The profiling endpoints are exposed on the same port as the HTTP REST API under the `/debug/pprof/` path. This could be useful for the extension developers.
+- [#3442](https://github.com/grafana/k6/pull/3442) The k6 now supports a new `--version` flag. It has the same output as `k6 version` command. Thanks, @ffapitalle!
+- [#3423](https://github.com/grafana/k6/pull/3423) The InfluxDB output got an environment variable `K6_INFLUXDB_PROXY` which allows specifying proxy. Thanks, @IvanovOleg!
+- [#3398](https://github.com/grafana/k6/pull/3398) The k6 cloud traces are enabled by default.
+- [#3400](https://github.com/grafana/k6/pull/3400) The Cloud output v2 is now the default version when the test streams metrics from a local test run via `-o cloud`.
```

Comment:
What do you think about mentioning that the key difference of the output is binary-based?

```suggestion
- [#3400](https://github.com/grafana/k6/pull/3400) The binary-based cloud output (a.k.a. cloud output v2) is now the default version when the test streams metrics from a local test run via `-o cloud`.
```